includes/emails/actions.php

```diff
@@ -25,7 +25,7 @@
  */
 function give_trigger_donation_receipt( $payment_id ) {
 	// Make sure we don't send a receipt while editing a donation.
-	if ( isset( $_POST['give-action'] ) && 'edit_payment' == $_POST['give-action'] ) {
```

Comment:
@DevinWalker I removed `give-action` comparison because I did not find usage of this in core and seems legacy code.

Reply:
Good idea.